src/odemis/util/linalg.py

```diff
@@ -230,3 +230,56 @@ def are_collinear(p1: Iterable[float], p2: Iterable[float], p3: Iterable[float])
     return abs((x2 - x1) * (y3 - y1) - (x3 - x1) * (y2 - y1)) < 1e-12 and \
         abs((x2 - x1) * (z3 - z1) - (x3 - x1) * (z2 - z1)) < 1e-12 and \
         abs((y2 - y1) * (z3 - z1) - (y3 - y1) * (z2 - z1)) < 1e-12
+
+
+def generate_triangulation_points(max_dist: float, given_area_coords: Iterable[float]) -> Iterable[tuple]:
+    """
+    Finds (x,y) positions on the given area. The points are evenly distributed from each other in the given area.
+    The distance between points is decided by maximum distance.
+
+    If the distance between points is more than the given maximum distance, the overview acquisition image might
+    result in blurry spots in the final acquisition image.
+
+    :param max_dist: the maximum distance allowed between two (x,y) positions.
+    :param given_area_coords: [xmin, ymin, xmax, ymax] the top right and bottom left (x,y) coordinates in meters.
+    :return: List of (x,y) coordinates in the given area.
+    """
+    # TODO Calculate points distribution for rectangular region where one side >> second side
+    # in the above case, points along the shorter side will be closer to each other and
+    # will not be evenly distributed throughout the given area
+    xmin, ymin, xmax, ymax = given_area_coords
+
+    # Avoid points exactly on the border of the given area, find points delta distance
+    # away from the border of the given area
+    delta_x = (1 / 20) * (xmax - xmin)
+    delta_y = (1 / 20) * (ymax - ymin)
```

Comment:
1/20th is a little bit arbitrary. What I'd suggest is to make it so that the delta is about (but a little bit smaller than) max_dist. Essentially, that consists in first computing the number of points needed for the whole area, and then removing the distance needed for 1 point. One advantage in addition to being less arbitrary is that it's exactly this algorithm that we use in Odemis for the placement of points in a grid in general (like for the AR positions on the SPARC, or the SEM beam positions in an RoI).
```
points_x = math.floor(length_x/max_dist) 
step_x = length_x / points_x
xmin = xmin + step_x/2
xmax = xmax - step_x/2
```
(the y dimension is the same)